### ISSUE TITLE
FsCheck QuickThrowOnFailure not surfacing exception messages on 3.0.0-beta1

### ISSUE THREAD

**Issue by eiriktsarpalis** [fscheck/FsCheck#591]
The following test is currently failing in the `fscheck3` branch:
```fsharp
open Swensen.Unquote
[<Fact>]
let ``QuickThrowOnFailure should surface exception message``() =
    try Check.QuickThrowOnFailure(fun i -> if abs i < 20 then true else failwith "bespoke error message")
    with e ->
        // both assertions currently failing
        test <@ e.Message.Contains "with exception:" @>
        test <@ e.Message.Contains "bespoke error message" @>

    do failwith "should have thrown an exception!"
```
This is a regression from 3.0.0-alpha5.

**Comment by kurtschelfthout** [comment]
Well spotted. Thanks for the report.

**Comment by kurtschelfthout** [comment]
Finally released in 3.0.0-beta2 - apologies dropped the ball on releasing this.
